test/c3/appframework/appframework_test.go

```diff
@@ -2806,7 +2806,95 @@ var _ = Describe("c3appfw test", func() {
 			shcAppSourceInfo.CrAppFileList = testenv.GetAppFileList(appListV2)
 			allAppSourceInfo = []testenv.AppSourceInfo{cmAppSourceInfo, shcAppSourceInfo}
 			testenv.AppFrameWorkVerifications(ctx, deployment, testcaseEnvInst, allAppSourceInfo, splunkPodAge, clusterManagerBundleHash)
+		})
+	})
+
+	Context("Clustered deployment (C3 - clustered indexer, search head cluster)", func() {
+		It("c3, integration, appframeworkc3, appframework: can deploy a C3 SVA and install a bigger volume of apps than the operator PV disk space", func() {
+
+			/* Test Steps
+			   ################## SETUP ####################
+			   * Upload 15 apps of 100MB size each to S3 for Indexer Cluster and Search Head Cluster for cluster scope
+			   * Create app sources for Cluster Manager and Deployer with cluster scope
+			   * Prepare and deploy C3 CRD with app framework and wait for the pods to be ready
+			   ######### INITIAL VERIFICATIONS #############
+			   * Verify Apps are Downloaded in App Deployment Info
+			   * Verify Apps Copied in App Deployment Info
+			   * Verify App Package is deleted from Operator Pod
+			   * Verify Apps Installed in App Deployment Info
+			   * Verify App Package is deleted from Splunk Pod
+			   * Verify bundle push is successful
+			   * Verify  apps are copied, installed on Search Heads and Indexers pods
+			*/
+
+			//################## SETUP ####################
+			// Download 15 apps around 100MB each (Total 1.4GB) to have total volume above Operator PV default size (1GB)
+			appVersion := "V1"
+			appList := testenv.PVTestApps
+			appFileList := testenv.GetAppFileList(appList)
+			err := testenv.DownloadFilesFromS3(testDataS3Bucket, s3PVTestApps, downloadDirPVTestApps, appFileList)
+			Expect(err).To(Succeed(), "Unable to download app files")
+
+			// Upload apps to S3 for Indexer Cluster
+			testcaseEnvInst.Log.Info("Upload 15 apps around 100MB each (Total 1.4GB) for Indexer Cluster to be above Operator PV default size (1GB)")
```

Comment:
One other alternative(a better alternative ?) is, to fill the local partition on the operator pod, so that we don't need to upload such a large file to S3:
Example:
`dd if=/dev/zero of=./gentoo_root.img bs=4k iflag=fullblock,count_bytes count=1G`

Reply:
Or could we test with a smaller PVC for this testcase?

Reply:
> Or could we test with a smaller PVC for this testcase?

We actually tried to edit the PV and set a smaller value but it seems 1G is the minimum here, lower values would be ignored.

Reply:
> One other alternative(a better alternative ?) is, to fill the local partition on the operator pod, so that we don't need to upload such a large file to S3: Example: `dd if=/dev/zero of=./gentoo_root.img bs=4k iflag=fullblock,count_bytes count=1G`

We will use another task to implement this suggestion later and can merge the current version of the test meanwhile.